job-scheduler/src/main/java/org/hawkular/metrics/scheduler/impl/SchedulerImpl.java

```diff
@@ -737,20 +766,20 @@ public boolean isAcquired() {
     }
 
     private static class JobLock {
-        private UUID jobId;
+        JobDetails jobDetails;
```

Comment:
Missing visibility modifier? Intentional? You have getJobDetails() in any case and that's seems to be used also.